src/main/java/com/tupilabs/testng/parser/TestNGParser.java

```diff
@@ -71,17 +72,23 @@ public Suite parse(File file) throws ParserException {
 		}
 
 		SAXParser parser = null;
+		List<Suite> suites;
 
 		try {
 			fileInputStream = new FileInputStream(file);
 			parser = factory.newSAXParser();
 			parser.parse(fileInputStream, handler);
 
-			suite = handler.getSuite();
-			if (suite == null) {
+			suites = handler.getSuite();
+			if (suites == null) {
```

Comment:
Think this is not necessary. Quite sure you are returning an empty list in the handler. We can also add a NotNull annotation (if we have the imports in the project) for future implementations keep that behaviour.